src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
 public class Duke {
+
+    private UI ui;
+
+   public String logo = " ____        _        \n"
```

Comment:
```suggestion
    private UI ui;

    public String logo = " ____        _        \n"
```

You have a missing space here